src/AclServiceProvider.php

```diff
@@ -23,10 +24,12 @@ public function boot()
             ], 'config');
         }
 
-        $this->app->make(DoctrineManager::class)->onResolve(function () {
+        $this->app->get(DoctrineManager::class)->onResolve(function () {
```

Comment:
I don't really understand why there is changes in this file?

you can still get container via `$this->app`